docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md

```diff
@@ -57,16 +57,19 @@ Horizontal autoscaling is available on all the Falcon cloud plans (OpenStack).
 
 Next to the provider, horizontal autoscaling does have a couple of additional requirements.
 
-### Supported CMS
+We will divide them between Hypernode-specific and Application-specific requirements
 
-Horizontal autoscaling is available for Magento2.
-To make use of Horizontal autoscaling, there are a couple of other requirements the Hypernode and the application should meet:
+### Hypernode Specific Requirements
 
-### Operating system
+#### Operating system
 
 - The operating system of the Hypernode should be Debian Bookworm
```

Comment:
Probably point them to support if they want to upgrade their hypernode

Reply:
Added link to customer support